components/Task/Task.js

```diff
@@ -0,0 +1,36 @@
+export default function Task({ task }) {
+  function prioColor() {
+    if (task.priority === "High") {
+      return "red";
+    } else if (task.priority === "Medium") {
+      return "yellow";
+    } else if (task.priority === "Low") {
+      return "green";
+    }
+  }
+
+  const today = new Date();
+  const dueDate = new Date(task.dueDate);
+  const pastDueDate = today >= dueDate;
+
+  return (
+    <div
+      style={{ border: "1px solid black", padding: "1rem", margin: "0.5rem" }}
+    >
+      <div style={{ display: "flex", gap: "0.5rem" }}>
+        <div
+          style={{
+            width: "20px",
+            height: "20px",
+            borderRadius: "50%",
+            background: prioColor(),
+          }}
+        ></div>
+        <div style={{ color: pastDueDate ? "red" : "black" }}>
+          {task.dueDate}
+        </div>
+      </div>
+      <h3>{task.title}</h3>
+    </div>
```

Comment:
Looks like a `div`-salad. 🙂 Please, use semantic html elements whenever possible. Your `TaskList` could be an `ul` and each task an `li` element. Inside of each `li` element can include a `section`.
For the due date you can also find a more semantic element than a `div`, e.g. a `p` element.

Additionally, please do not use inline styles but styled-components instead. This makes the code more readable and maintainable. You can pass the `priority` and `pastDueDate` as a prop to your styled component and change the color accordingly. With the change to styled components, you could therefore also get rid of the `prioColor` function.